models/deeplab/evaluate.py

```diff
@@ -25,47 +25,47 @@ def __init__(self, args):
         # Define Saver
         self.saver = Saver(args)
         self.saver.save_experiment_config()
-        
+
         # Define transforms and Dataloader
-        deeplab_collate_fn = None
-        transform = None
-        test_dataset = build_test_dataloader(args.dataset, args.data_root, transform)
+        if args.switch_r_and_b:
+            transform = lambda x: x[[2,1,0],:,:].copy()
+        else:
+            transform = lambda x: x[:3,:,:].copy()
```

Comment:
Let's use `x.permute` for this. In your TileDataset, convert the numpy array to a Tensor and then use permute.

Reply:
Also, how come this is copying the array?